molecule/prepare.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Prepare
```

Comment:
I'm not familiar enough with molecule to know how standard this is, but to me, this kind of preparation should be directly in the docker image.

Reply:
I don't really get where you'd like to move it :sweat_smile:.
We can't put it in the create.yml as the inventory is not yet generated. Should we maintain our own systemd images ? I'm not sure it worths the pain.